config/local.json-dist

```diff
@@ -1,6 +1,16 @@
 {
-  "env": "dev",
+  "env": "development",
   "log": {
     "level": "debug"
+  },
+  "basket":{
+    "sqs": {
+       "region": {
+         "default": "local"
```

Comment:
there should be no `default` property, just the value: http://v14d.com/i/5aba99c75c39c.jpg